### PR TITLE
fix default values in configuration documentation

### PR DESCRIPTION
**Pull request by wkloucek**
## Description
update flaex to fix default values in configuration documentation, see https://github.com/owncloud/flaex/pull/10

### CODE REVIEW

File: .bingo/README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-# Project Development Dependencies
+# Project Development Dependencies.
```
Comment (refs) [inline .bingo/README.md:1]:
```suggestion
# Project Development Dependencies
```

File: .bingo/README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 This is directory which stores Go modules with pinned buildable package that is used within this repository, managed by https://github.com/bwplotka/bingo.
 
-*   Run `bingo get` to install all tools having each own module file in this directory.
-*   Run `bingo get <tool>` to install <tool> that have own module file in this directory.
-*   For Makefile: Make sure to put `include .bingo/Variables.mk` in your Makefile, then use $(<upper case tool name>) variable where <tool> is the .bingo/<tool>.mod.
-*   For shell: Run `source .bingo/variables.env` to source all environment variable for each tool.
-*   For go: Import `.bingo/variables.go` to for variable names.
-*   See https://github.com/bwplotka/bingo or -h on how to add, remove or change binaries dependencies.
+* Run `bingo get` to install all tools having each own module file in this directory.
```
Comment (refs) [inline .bingo/README.md:5]: remark is a REALLY tedious piece of software: https://github.com/remarkjs/remark-lint/issues/178

Comment (refs) [reply .bingo/README.md:5]:
```suggestion
*   Run `bingo get` to install all tools having each own module file in this directory.
```

File: .bingo/README.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+* Run `bingo get <tool>` to install <tool> that have own module file in this directory.
```
Comment (refs) [inline .bingo/README.md:6]:
```suggestion
*   Run `bingo get <tool>` to install <tool> that have own module file in this directory.
```

File: .bingo/README.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+* For Makefile: Make sure to put `include .bingo/Variables.mk` in your Makefile, then use $(<upper case tool name>) variable where <tool> is the .bingo/<tool>.mod.
```
Comment (refs) [inline .bingo/README.md:7]:
```suggestion
*   For Makefile: Make sure to put `include .bingo/Variables.mk` in your Makefile, then use $(<upper case tool name>) variable where <tool> is the .bingo/<tool>.mod.
```

File: .bingo/README.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+* For shell: Run `source .bingo/variables.env` to source all environment variable for each tool.
```
Comment (refs) [inline .bingo/README.md:8]:
```suggestion
*   For shell: Run `source .bingo/variables.env` to source all environment variable for each tool.
```

File: .bingo/README.md [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
+* For go: Import `.bingo/variables.go` to for variable names.
```
Comment (refs) [inline .bingo/README.md:9]:
```suggestion
*   For go: Import `.bingo/variables.go` to for variable names.
```

File: .bingo/README.md [hunk 7]
```diff
@@ -1,14 +1,14 @@
...
+* See https://github.com/bwplotka/bingo or -h on how to add, remove or change binaries dependencies.
```
Comment (refs) [inline .bingo/README.md:10]:
```suggestion
*   See https://github.com/bwplotka/bingo or -h on how to add, remove or change binaries dependencies.
```

File: .bingo/README.md [hunk 8]
```diff
@@ -1,14 +1,14 @@
...
 
 ## Requirements
 
-*   Go 1.14+
+* Go 1.14+
```
Comment (refs) [inline .bingo/README.md:14]:
```suggestion
*   Go 1.14+
```

**Review by refs — CHANGES_REQUESTED** [review]

**Comment by wkloucek** [comment]
@refs `.bingo/README.md` is autogenerated by https://github.com/bwplotka/bingo. This means it will be reformatted on the next update `bingo get ...` :-( But I reverted it now ...

**Review by refs — APPROVED** [review]

**Pull request merged**